test/integration/utils/compose.go

```diff
@@ -98,14 +98,14 @@ func (c *ComposeConf) Stop() ([]byte, error) {
 	if c.RemoveRebuildImages {
 		args = append(args, "--rmi", "all")
 	}
-	runCmd := exec.Command("docker-compose", args...)
+	runCmd := exec.Command("compos", args...)
```

Comment:
I think we have a typo here

```suggestion
	runCmd := exec.Command("compose", args...)
```

Reply:
😭 that would definitely help. Thanks!